beacon_node/http_api/src/task_spawner.rs

```diff
@@ -103,6 +111,17 @@ impl<E: EthSpec> TaskSpawner<E> {
         self,
         priority: Priority,
         func: impl Future<Output = Result<Response, warp::Rejection>> + Send + Sync + 'static,
+    ) -> Response {
```

Comment:
> /// Executes an async task which may return a `warp::Rejection`.

The method doc above should probably be updated now the original function has moved to  `spawn_async_with_rejection_no_conversion` method.

Reply:
Good catch, updated in https://github.com/sigp/lighthouse/pull/4595/commits/7c04d902e19a13bfe39ac8bb1923022144b0fc26